src/services/subject.js

```
@@ -43,6 +44,28 @@
+  },
+
+  updateSubject: async (subjectId, updateData) => {
+    const subject = await Subject.findById(subjectId);
+
+    if (!subject) {
+      const error = new Error('Subject not found.');
+      error.status = 404;
+      throw error;
+    }
+
+    const validatedData = await validateCategoryOnUpdate(subjectId, updateData);
```

ESLint

[error] Extra semicolon.

(semi)